app/src/main/java/com/swent/assos/ui/components/EventItem.kt

```diff
@@ -44,10 +42,7 @@ fun EventItem(event: Event, navigationActions: NavigationActions, asso: Associat
                 navigationActions.navigateTo(Destinations.EVENT_DETAILS.route + "/${event.id}")
               }) {
         Column(
-            modifier =
-                Modifier.width(200.dp).padding(vertical = 0.dp).clickable(true) {
-                  navigationActions.navigateTo(Destinations.EVENT_DETAILS.route + "/${event.id}")
-                },
+            modifier = Modifier.width(200.dp).padding(vertical = 0.dp),
```

Comment:
are we sure it is card that we want to click not the column?

Reply:
Yes and it resolved the click bug